config/locales/en.yml

```diff
@@ -39,6 +39,8 @@ en:
       edit?: "Sorry, you can only edit case contacts that you have logged."
       update?: "Sorry, you can only edit case contacts that you have logged."
   forms:
+    casa_admin:
+      mark_inactive_warning: "WARNING: Marking an admin inactive will make them unable to login. Are you sure you want to do this?"
```

Comment:
Will admins get an email informing them of their account deactivation, like volunteers do? We will want to make a github issue for it if so @Zrrrpy